lib/user-methods.js

```diff
@@ -275,6 +275,32 @@ function _authenticate(userConstructor, server, json, callback) {
     }
 }
 
+function _updateAccount(userConstructor, server, json) {
+    const url = append_url(server, 'auth/password/updateAccount');
+    const options = {
+        method: 'POST',
+        body: JSON.stringify(json),
+        headers: postHeaders,
+        open_timeout: 5000
+    };
+
+    const promise = performFetch(url, options)
```

Comment:
No need to assign a new local for that - you can simply return.

Reply:
hmm, I tried to follow the same logic as in https://github.com/realm/realm-js/blob/d0b0adabc92bc9da2da9d881a5d099a8ec4596ce/lib/user-methods.js#L243

Reply:
In `_authenticate`, the promise is assigned to a local because there may be a callback that needs to get invoked. In the current case, there's no such thing, so there's no point to assign it just to return it on the next line :)